.tekton/central-db-pull-request.yaml

```
@@ -0,0 +1,434 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/stackrox/stackrox?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "500"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request"
```

Optional suggestion.

```suggestion
    pipelinesascode.tekton.dev/on-event: '[pull_request]'
    pipelinesascode.tekton.dev/on-target-branch: '[*]'
```

This should give the same effect but will be uniform with `roxctl` pipeline.
Either way we decide, let's please keep all pipelines similar.

---

Discussed in weekly meeting to have a reconcile at the end of all onboardings: https://issues.redhat.com/browse/ROX-20814. 
I would postpone it to that task.